ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -494,19 +494,13 @@ objects:
           for server in $(openstack server list -c Name -f value | grep $CLUSTER_NAME | sort); do
               echo -e "\n$ openstack server show $server" >>${ARTIFACT_DIR}/openstack_nodes.log
               openstack server show $server >>${ARTIFACT_DIR}/openstack_nodes.log
+              if [[ "$server" == *"bootstrap"* ]]; then
+                openstack console log show $server &>${ARTIFACT_DIR}/bootstrap/nova.log
```

Comment:
There is no `bootstrap` directory in the [build artifacts](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/11446/rehearse-11446-pull-ci-openshift-baremetal-runtimecfg-master-e2e-openstack/1300705536727060480/artifacts/e2e-openstack/). Is this conditional meant to work in failed clusters, where the bootstrap node is not deleted?